doc/start/quick-ceph-deploy.rst

```diff
@@ -26,172 +26,136 @@ are in this directory when executing ``ceph-deploy``.
-   it must be reachable by its hostname. If necessary, modify ``/etc/hosts``
-   to add the name of the admin host.
+#. Deploy a manager daemon.::
 
-#. Ensure that you have the correct permissions for the
-   ``ceph.client.admin.keyring``. ::
+     ceph-deploy mgr create node1
 
-	sudo chmod +r /etc/ceph/ceph.client.admin.keyring
+#. Add three OSDs. For the purposes of these instructions, we assume you have an
+   unused disk in each node called ``/dev/vdb``.  *Be sure that whatever device you use is not currently in use and does not contain any important data.*
 
-#. Check your cluster's health. ::
+     ceph-deploy osd create {ceph-node}:{device}
 
-	ceph health
-
-   Your cluster should return an ``active + clean`` state when it
-   has finished peering.
+   For example::
 
+     ceph-deploy osd create node1:vdb node2:vdb node3:vdb
 
-Operating Your Cluster
-======================
+#. Check your cluster's health. ::
 
-Deploying a Ceph cluster with ``ceph-deploy`` automatically starts the cluster.
-To operate the cluster daemons with Debian/Ubuntu distributions, see
-`Running Ceph with Upstart`_.  To operate the cluster daemons with CentOS,
-Red Hat, Fedora, and SLES distributions, see `Running Ceph with sysvinit`_.
+     ssh node1 sudo ceph health
 
-To learn more about peering and cluster health, see `Monitoring a Cluster`_.
-To learn more about Ceph OSD Daemon and placement group health, see
-`Monitoring OSDs and PGs`_. To learn more about managing users, see
-`User Management`_.
+   Your cluster should report ``HEALTH_OK``.  You can view a more complete
+   cluster status with::
 
-Once you deploy a Ceph cluster, you can try out some of the administration
-functionality, the ``rados`` object store command line, and then proceed to
-Quick Start guides for Ceph Block Device, Ceph Filesystem, and the Ceph Object
-Gateway.
+     ssh node1 sudo ceph -s
 
 
 Expanding Your Cluster
 ======================
 
-Once you have a basic cluster up and running, the next step is to expand
-cluster. Add a Ceph OSD Daemon and a Ceph Metadata Server to ``node1``.
-Then add a Ceph Monitor to ``node2`` and  ``node3`` to establish a
-quorum of Ceph Monitors.
+Once you have a basic cluster up and running, the next step is to
+expand cluster. Add a Ceph Metadata Server to ``node1``.  Then add a
+Ceph Monitor and Ceph Manager to ``node2`` and ``node3`` to improve reliability and availability.
 
 .. ditaa::
            /------------------\         /----------------\
            |    ceph-deploy   |         |     node1      |
            |    Admin Node    |         | cCCC           |
            |                  +-------->+   mon.node1    |
-           |                  |         |     osd.2      |
-           |                  |         |   mds.node1    |
+           |                  |         |     osd.0      |
+           |                  |         |   mgr.node1    |
+	   |                  |         |   mds.node1    |
```

Comment:
nit, would be better if we can use spaces here even sphinx will remove the leading spaces and tabs before feeding this diagram to ditaa.